commons/ava_testsuite/rpc_manager.go

```diff
@@ -0,0 +1,147 @@
+package ava_testsuite
+
+import (
+	"github.com/kurtosis-tech/ava-e2e-tests/commons/ava_default_testnet"
+	"github.com/kurtosis-tech/ava-e2e-tests/gecko_client"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+	"time"
+)
+
+const (
+	GENESIS_USERNAME = "genesis"
+	GENESIS_PASSWORD = "genesis34!23"
+	ACCEPTED_STATUS = "Accepted"
```

Comment:
Let's add a bit more info about what exactly is accepted (a transaction? a user? etc.). General pattern I try to follow: rather than describing what the constant's value is, you describe what the constant's semantic meaning is.